src/components/Providers/MessageRenderer.tsx

```diff
@@ -126,6 +131,19 @@ export const MessageRenderer = ({ children }: MessageRendererWrapperProps) => {
     }
   };
 
+  const getChatBadge = ({
+    user,
+    TagWrapper
+  }: GetChatBadgeArgs): React.ReactElement<
+    any,
+    string | React.JSXElementConstructor<any>
+  > | null => {
```

Comment:
this isn't necessary, ts automatically gets the correct return type and it works
```suggestion
  }: GetChatBadgeArgs) => {
```